### DISCUSSION TITLE
How to run client or server from vs2022

### DISCUSSION THREAD

**Discussion by Rinntrah** [ReactiveDrop/reactivedrop_public_src#813] [Q&A]
Instructions unclear. Running project ends with following error:
![image](https://github.com/user-attachments/assets/f0feb061-b903-4d38-8061-dae500df0699)

How to run the game or server?
Where is exe?

Instructions from git repository:
`How to compile the project
Download Visual Studio 2022 Community (https://visualstudio.microsoft.com/downloads/)
While installing Visual Studio 2022 select the 'Game development with C++' workload
Open reactivedrop_vs13.sln using Visual Studio
Select configuration Debug or Release
Build
Resulting DLLs will be placed into reactivedrop\bin`

**Comment by mithrand0** [comment]
You need to download the game from Steam, and replace the client/server.dll in the game folder (reactivedrop/bin/client.dll, reactivedrop/bin/server.dll)

https://store.steampowered.com/app/563560/Alien_Swarm_Reactive_Drop/

Be aware you cannot join public lobbies with a custom client.dll, since those check for an official client.
